spec/features/navigation_spec.rb

```diff
@@ -32,6 +56,70 @@
         expect(page).to have_content("Cart: 0")
       end
 
+      visit '/cart'
+
+      within 'nav' do
+        expect(page).to have_content("Cart: 0")
+      end
+
+      visit '/login'
+
+      within 'nav' do
+        expect(page).to have_content("Cart: 0")
+      end
+
+      visit '/register'
+
+      within 'nav' do
+        expect(page).to have_content("Cart: 0")
+      end
     end
   end
+  # describe "As a Default user" do
+  #   it "I see all links, plus profile and logout links. I do NOT see login or register links" do
```

Comment:
This is test is for the next user story, right?

Reply:
correct. User story 3 :)